### ISSUE TITLE
GetWallTime Broken

### ISSUE THREAD

**Issue by spaulaus** [pixie16/pixie_scan#8]
The method DetectorDriver::GetWallTime() is broken due to being used with a firmware specific (SHE) implementation. This requires that VSN = 1000 be set to properly initialize the wall time. For newer firmwares we never receive this VSN and so this method returns 0.

**Comment by ksmith0** [comment]
I do not believe this is a firmware issue. Wall clock was removed in early version of poll2 and never restored. The clock and can 1000 are inserted by poll into the data stream. I can restore this functionality if it is needed.

**Comment by spaulaus** [comment]
Ah....it should be restored.

**Comment by ksmith0** [comment]
I'll fix it this week.

**Comment by ksmith0** [comment]
Can we cross link this with an issue in PixieSuite2?
